torch/library.py

```diff
@@ -15,6 +15,11 @@
 # prim is reserved by TorchScript interpreter
 _reserved_namespaces = ['prim']
 
+def fallthrough_kernel():
+    """
+    A dummy function to pass to ``Library.impl`` in order to register a fallthrough.
+    """
+    return
```

Comment:
This should never be called right? Raising a NotImplementedError() sounds better?